src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -112,7 +112,7 @@ class AddUserProfile extends Component {
           <Row>
             <Col md="12">
               <Form>
-                <Row>
+                <Row className='userProfileAddRow'>
```

Comment:
I've noticed that the class name 'userProfileAddRow' is quite descriptive, but it could be a bit long and potentially confusing with similar class names. To maintain simplicity and improve readability, I would suggest shortening it. A more concise class name such as 'user-row' would still convey the key idea that this is a row in a user form, while being more succinct.

Reply:
Thanks for your suggestion! I have shorten the className.